lti_consumer/lti_xblock.py

```diff
@@ -142,6 +142,7 @@ class LaunchTarget:
 
 
 @XBlock.needs('i18n')
+@XBlock.needs('rebind-user')
```

Comment:
It's `rebind_user`.

Otherwise, this will throw `xblock.exceptions.NoSuchServiceError: Service 'rebind_user' was not requested.`.